pkgs/servers/nosql/influxdb2/default.nix

```diff
@@ -0,0 +1,128 @@
+{ buildGoModule
+, buildGoPackage
+, fetchFromGitHub
+, go-bindata
+, lib
+, llvmPackages
+, mkYarnPackage
+, pkg-config
+, rustPlatform
+, ... }:
```

Comment:
Is the ellipsis necessary?

Reply:
Huh, I guess not! Force of habit from my NixOS configs. Removed.